index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main-doc">
+      <nav class="nav-bar" id="navbar">
+        <header>Web Developer</header>
+        <ul>
+          <li><a class="nav-link" href="#Skills">Skills</a></li>
+          <li><a class="nav-link" href="#Cooperative">Cooperative</a></li>
+          <li><a class="nav-link" href="#Fast_learn">Fast learn</a></li>
+          <li><a class="nav-link" href="#Logical">Logical</a></li>
+          <li>
+            <a class="nav-link" href="#Positive_attitude">Positive attitude</a>
+          </li>
+        </ul>
+      </nav>
+      <section class="main-section" id="Skills">
+        <header>Skills</header>
+        <p></p>
+
+        <p></p>
+        <code></code>
+        <ul>
+          <li></li>
+        </ul>
```

Comment:
please remove or add something to this list

Reply:
- [ ] Good job so far, :+1:  I think you can improve this message by adding a short explanation, a "why", the requested change needed to be made. This will make your comment more professional. :smile_cat: